internal/usagegen/usagegen.go

```diff
@@ -60,6 +60,7 @@ func Generate(ctx context.Context, customerID, lang, schemaPath, header, token,
 		generate.WithRunLocation("cli"),
 		generate.WithGenVersion(strings.TrimPrefix(changelog.GetLatestVersion(), "v")),
 		generate.WithAllowRemoteReferences(),
+		generate.WithForceGeneration(),
```

Comment:
We need this because if there is no detected SDK change in the gen.lock codeSamples won't generate. But we always want them to generate idempotently.